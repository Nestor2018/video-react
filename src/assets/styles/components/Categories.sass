.categories__title
  background: $purple-brand
  color: $white-brand
  font-size: 16px
  margin-top: -16px
  position: absolute
  padding-left: 30px
  width: 100%
