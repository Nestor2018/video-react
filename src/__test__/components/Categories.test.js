import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';

import Categories from '../../components/Categories';

describe('<Categories/>', () => {
	const categories = mount(<Categories />);
	test('Render del componente categories', () => {
		expect(categories.length).toEqual(1);
	});
});

describe('Categories Snapshot', () => {
	const categories = create(<Categories />);
	test('Comprobar la UI del componente Categories', () => {
		expect(categories.toJSON()).toMatchSnapshot();
	});
});
