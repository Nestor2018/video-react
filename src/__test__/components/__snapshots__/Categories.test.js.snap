// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories Snapshot Comprobar la UI del componente Categories 1`] = `
<div
  className="Categories"
>
  <h3
    className="categories__title"
  />
</div>
`;
